python/nav/web/auth/utils.py

```diff
@@ -87,3 +90,21 @@ def authorization_not_required(fullpath):
         if fullpath.startswith(url):
             _logger.debug('authorization_not_required: %s', url)
             return True
+
+
+def get_number_of_accounts_with_password_issues() -> int:
+    number_of_accounts_with_password_issues = cache.get(
+        PASSWORD_ISSUES_CACHE_KEY, default=None
+    )
+    if number_of_accounts_with_password_issues is None:
+        number_of_accounts_with_password_issues = 0
+        for account in Account.objects.all():
+            if not account.is_default_account() and (
+                account.has_plaintext_password()
+                or account.has_old_style_password_hash()
+                or account.has_deprecated_password_hash_method()
```

Comment:
See my comment on #3407. Now you have a third location that needs to change if the list of testable password deficiencies is modified.